index.html

```diff
@@ -1,10 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/navbar.css">
     <meta charset="UTF-8">
-    <title>Better than Apple</title>
+    <title>let fake = (🍎) =>🍋</title>
 </head>
 <body>
-        
+    <main id="hero" class="container">
+        <header class="inside-container">
+            <nav class="navbar">
+                <ul class="nav-menu glossy-effect">
+                    <li class="box">
+                        <a class="" href="#">
+                        <img class="logo" src="https://upload.wikimedia.org/wikipedia/commons/9/9a/Logo_apple_pnh.png" alt=""></a>
+                    </li>
+                    <li class="store box"><a class=""  href="#">Store</a></li>
+                    <li class="box"><a class="mac"    href="#">Mac</a></li>
+                    <li class="box"><a class="ipod"   href="#">Ipod</a></li>
+                    <li class="box"><a class="iphone" href="#">Iphone</a></li>
+                    <li class="box"><a class="ipad"   href="#">Ipad</a></li>
+                    <li class="box"><a class="itunes" href="#">Itunes</a></li>
+                    <li class="box">
+                        <a class="support" href="#">Support</a></li>
+                    <li class="clear-margin">
+                        <div class="search-bar">
+                            <span class="magnify"></span>
+                            <input id="search-box" type="text" name="search">
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+            <div id="hero-textbox" >
+                <div id="obama">
+                    <div class="title"><a href="#"></a></div>
+                    <div class="subtitle"><a href="#"></a></div>
+                    <div class="call-to-action"><a href="#"></a></div>
+                </div>
+            </div>
+        </header>
+    </main>
+    
+    <div id="showcase">
+        <img src="https://archive.fo/UW4oR/4199f096e33e638ba9aeb6c6dec3a3e64794d02c.jpg" alt="" />
```

Comment:
The img tags should be wrapped in `<a>` tags do this before merging.

Reply:
Fixed , Thank you!